bamboo-specs/bamboo.yaml

```diff
@@ -276,13 +292,89 @@ Release ORCA Code:
         set -ex
         bin/build_release_package.sh
       description: Create Release Files and Release in GitHub
+
+#job for deploying cumulus and orca
+Deploy RDS cluster:
+  key: DRC
+  other:
+    clean-working-dir: true
+  docker:
+    image: maven.earthdata.nasa.gov/cumulus_orca
+    volumes:
+      ${bamboo.working.directory}: ${bamboo.working.directory}
+      ${bamboo.tmp.directory}: ${bamboo.tmp.directory}
+  tasks:
+  - checkout:
+      force-clean-build: 'true'
+      description: Checkout Default Repository
+  - script:
+      interpreter: SHELL
+      scripts:
+      - |-
+        #!/bin/bash
+        chmod +x bin/deployment-rds.sh
+        bin/deployment-rds.sh
+#job for deploying cumulus and orca
+Deploy cumulus and orca:
+  key: DCO
+  other:
+    clean-working-dir: true
+    # Some plugin configurations are not supported by YAML Specs
+  docker:
+    image: maven.earthdata.nasa.gov/cumulus_orca
+    volumes:
+      ${bamboo.working.directory}: ${bamboo.working.directory}
+      ${bamboo.tmp.directory}: ${bamboo.tmp.directory}
+  tasks:
+  - checkout:
+      force-clean-build: 'true'
+      description: Checkout Default Repository
+
+  - script:
+      interpreter: SHELL
+      scripts:
+      - |-
+        #!/bin/bash
+        chmod +x bin/deployment-cumulus-orca.sh
+        bin/deployment-cumulus-orca.sh
+
 variables:
-  ORCA_VERSION: 4.0.0
+  DEPLOYMENT: sandbox
+  ORCA_VERSION: 0.0.1
   RELEASE_FLAG: 'false'
   REPORT_BUILD_STATUS: 'true'
   SECRET_GITHUB_EMAIL: BAMSCRT@0@0@HYUkicwfEqt17JZaZuGYmtmRHrnDySiT+kQkBQa7/Rg7IQgl1lUvbncMUu0Wx2k1ayHojNkh2TdOlF0kJ5tw6Q== #encrypted
   SECRET_GITHUB_TOKEN: BAMSCRT@0@0@nPxyXQXMyE6zm3B1oI8Goma78s0H1fJD4vHy6+dpBQJpjDI2ekbcvCAOI8AyjYiILPhqi2w4fT0fx9MFVXxz2JPTp5uPNzG3GM/8XSUgvYo= #encrypted
   SECRET_GITHUB_USER: BAMSCRT@0@0@fFtitNITDsAjJqK2ZXjhnBZ96z2775QLzGR1Xz557gSl8HxxBaTD3Fbmp+HUbpfx  #encrypted
+  AWS_DEFAULT_REGION: us-west-2
```

Comment:
Several of these we could just lookup or programmatically construct instead of putting these all in as environment variables. For example the following could easily be constructed:
 - TFSTATE_LOCK_TABLE: ${PREFIX}-tf-locks
 - TFSTATE_BUCKET: ${PREFIX}-tf-state
 - ORCA_REPORTS_BUCKET_NAME: ${PREFIX}-orca-primary
 
Another group we could lookup either using terraform or the aws cli in our scripts
Using `terraform output -json > ./infrastructure.json` you could use a utility like `jq` to parse the json and pull out the values needed. This would apply to your settings for:
- RDS_SECURITY_GROUP
- RDS_USER_ACCESS_SECRET_ARN
- RDS_CLUSTER_ID
- DB_HOST_ENDPOINT

Using the aws cli you could lookup the values for your other variables like the following:
- AWS_SUBNET_ID1
- AWS_SUBNET_ID2 - `aws ec2 describe-subnets | jq -r '.Subnets | .[] |  select (.Tags | .[] | .Value | contains ("Private application ")) | .SubnetId '`
-  VPC_ID - `aws ec2 describe-vpcs | jq -r '.Vpcs | to_entries | .[] | .value.VpcId'`
-  AWS_ACCOUNT_ID - ` aws sts get-caller-identity | jq -r '.Account'`

Note that we could also leverage terraform `data` to look these up as part of deployment. 


Others we could/should hardcode in our tfvars files like:
  - RDS_ENGINE_VERSION: 10.18 *Should double check this value for Aurora serverless versions* 
  - ROLE_BOUNDARY: NGAPShRoleBoundary

This work could be rolled into some of the environment work or we should create a card for these updates and enhancements unless you feel this is something you could easily do in this PR.

Reply:
For 
TFSTATE_LOCK_TABLE: ${PREFIX}-tf-locks
TFSTATE_BUCKET: ${PREFIX}-tf-state
ORCA_REPORTS_BUCKET_NAME: ${PREFIX}-orca-primary,
those are already pulled dynamically and we don't need to set manually. Forgot to remove them from here.


For the other variables that can be pulled using cli or terraform, I suggest creating another card in backlog since I won't be able to test them now in aws but that looks doable.

Reply:
created https://bugs.earthdata.nasa.gov/browse/ORCA-464